### PR TITLE
mh_*: add page

### PR DESCRIPTION
**Pull request by florianschanda**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** current master but doesn't matter really

### CODE REVIEW

**Comment by florianschanda** [comment]
I am creating this on request: https://github.com/florianschanda/miss_hit/issues/241

File: pages/common/mh_copyright.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# mh_copyright
+
+> Adjust copyright headers for MATLAB or Octave code.
+>
+> More advanced options are available, but they require
+> configuration. Please refer to:
+> <https://florianschanda.github.io/miss_hit/configuration.html>.
+>
+> More information: <https://misshit.org/>.
```
Comment (marchersimon) [inline pages/common/mh_copyright.md:9]: We only allow at most 3 description lines to keep them clean.
```suggestion
> Adjust copyright headers for MATLAB or Octave code.
> More information: <https://misshit.org/>.
```

File: pages/common/mh_style.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Auto-fix format problems in a file:
+
+`mh_style --fix {{filename.m}}`
+
+- Process an entire directory:
+
+`mh_style {{path/to/<placeholder>}}`
+
+- Process and fix everything in the current directory
```
Comment (marchersimon) [inline pages/common/mh_style.md:26]:
```suggestion
- Process and fix everything in the current directory:
```

File: pages/common/mh_trace.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# mh_trace
+
+> Extract tracing tags from MATLAB or Octave code.
+> More information: <https://misshit.org/>.
+
+- Add this comment to tag a function/block/class/file:
+
+`%| pragma Tag ({{"my_tag"}});`
+
+- Add this comment to exclude a function/block/class/file
```
Comment (marchersimon) [inline pages/common/mh_trace.md:10]:
```suggestion
- Add this comment to exclude a function/block/class/file:
```

**Review by marchersimon — COMMENTED** [review]
This will fix the remaining errors.
Also, please try to avoid force-pushing. It makes the review process a little harder and we squash before merging anyways.

**Comment by florianschanda** [comment]
Yep, sorry long radio silence. I am going to look at it today!

**Pull request closed without merging**